package.json

```diff
@@ -10,5 +10,8 @@
     "lint": "cd frontend && npm run lint",
     "postinstall": "npm run build && npm run deploy",
     "start": "cd frontend && npm start"
+  },
+  "dependencies": {
+    "bootstrap": "^4.3.1"
```

Comment:
Our frontend dependencies are all in `frontend/package.json`, and bootstrap is already provided. So I think you can back out the changes to this and package-lock.json in the root folder.